docs/TEAM.md

```diff
@@ -25,6 +25,9 @@ Micheal Aderinto, Manchester UK. Hoping this will be by first step in the world
 - [Nishank Navelkar](https://www.github.com/nishanknavelkar)
 Im from India, Basic knoweledge of Python, C, C++, HTML, CSS,  Electrical engineer, Here to learn and Contribute.
 
+- [Manohar Rajaram](https://github.com/manohar52)
+I am from India but currently pusruing my Masters in computer science in US. I am quite new to Javascript. I have some experience in HTML, CSS and PHP.
```

Comment:
pursuing wrong spelling

Reply:
Corrected the spelling.. thank you for that